airbyte-integrations/connectors/source-typeform/acceptance-test-config.yml

```diff
@@ -3,25 +3,30 @@ acceptance_tests:
   spec:
     tests:
     - spec_path: "source_typeform/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.12"
```

Comment:
this should be previous version - 0.2.0

Reply:
tests are skipped properly only with version 0.1.12, keeping it as is for now

Reply:
checked again, 0.2.0 also works